docs/apis-hook.md

```diff
@@ -4,4 +4,149 @@ sidebar_position: 4
 
 # APIs hook
 
-This page will discuss - How to interact with the ReactPlay data using custom hook.
+In React Play, we work with APIs by using our custom-made react hooks. They are as follows:
+
+1. [useCacheResponse](#usecacheresponse)
+1. [useContributors](#usecontributors)
+1. [useFeaturedPlays](#usefeaturedplays)
+1. [useFetch](#usefetch)
+1. [useGetPlays](#usegetplays)
+1. [useGitHub](#usegithub)
+1. [useLikePlays](#uselikeplays)
+1. [useLocalStorage](#uselocalstorage)
+
+In this section, we will discuss each of them in detail.
+
+## useCacheResponse
+
+As the name suggests, this hook is used to create and maintain a cached version of the data.
+
+```JavaScript
+  const [getCacheResponse, setCacheResponse] = useCacheResponse();
+  const response = axios(API_URL);
+  setCacheResponse(FILTER_DATA_RESPONSE, response);
+  // ....
+ const isCachedResponse = getCacheResponse(FILTER_DATA_RESPONSE);
+```
+
+`useCachedResponse()` returns a getter and a setter function. In the above example, we are setting the `FILTER_DATA_RESPONSE` value based on the API response. This value is now stored as a cached response. We can then use the getter function to access this cached value by passing the same key of `FILTER_DATA_RESPONSE`
+
+## useContributors
+
+This custom hook helps in retrieving the names of the contributors in [react play](https://github.com/reactplay/react-play) repository on GitHub. `useContributors` takes one parameter `sorted` which is a boolean value. If `true` is passed, the result is sorted in the ascending order of the number of contributions by the contributor. This function returns an object with `{ data, error, isLoading }`.
```

Comment:
`If `true` is passed, the result is sorted in the ascending order of the number of contributions by the contributor.` - this sentence is confusing.

Reply:
Initially, I had incorrectly understood the sorting approach. I have corrected this sentence to as follows:
"If `true` is passed, the result is a sorted list of contributors based on the number of contributions (highest to lowest)."
Let me know if it can be simplified further.